src/index.js

```diff
@@ -13,10 +13,14 @@ import max from './max'
 import isValid from './isValid'
 import convertTo from './convertTo'
 import unixTime from './unixTime'
+import fromTime from './fromTime'
+import of from './of'
 
 const clone = date => new Date(date.getTime())
 
 module.exports = {
+  of,
```

Comment:
yup, but I expect a few iterations, befor we settle on how the new api should be